### PR TITLE
Update VVL docs for Khronos layer

### PR DESCRIPTION
**Pull request by mark-lunarg**
- layers/README.md
- layers/vk_layer_settings_file.txt
- docs/khronos_validation_layer.md
- docs/gpu_validation.md
- CONTRIBUTING.md

### CODE REVIEW

File: docs/gpu_validation.md [hunk 1]
```diff
@@ -275,22 +279,21 @@ More detail is found in the discussion of the individual hooked functions below.
...
 calls outside of the application's API usage to create resources and perform its required operations
-inside of the core validation layer.
+inside of the validation layer.
 These calls are not routed up through the top of the loader/layer/driver call stack via the loader.
-Instead, they are simply dispatched via the core validation layer's dispatch table.
+Instead, they are simply dispatched via the containing layer's dispatch table.
 
-These calls therefore don't pass through core validation or any other validation layers that may be
-loaded/dispatched prior to code validation.
+These calls therefore don't pass through any validation checks that occure before the gpu validation checks are run.
```
Comment (mikew-lunarg) [inline docs/gpu_validation.md]: occur

File: docs/gpu_validation.md [hunk 2]
```diff
@@ -299,22 +302,22 @@ This doesn't present any particular problem, but it does raise some issues:
   To address this, the code can "just" be written carefully so that it is "valid" Vulkan,
   which is hard to do.
 
-  Or, this code can be checked by loading a core validation layer with
+  Or, this code can be checked by loading a khronos validation layer with
```
Comment (mikew-lunarg) [inline docs/gpu_validation.md]: capitalize Khronos?

File: docs/gpu_validation.md [hunk 3]
```diff
@@ -299,22 +302,22 @@ This doesn't present any particular problem, but it does raise some issues:
...
   Can't use the exposed mechanisms because we probably don't want it on twice.
-  * Rename this layer binary to something else like "core_validation2" to keep it apart from the
-  "normal" core validation.
+  * Rename this layer binary to something else like "khronos_validation2" to keep it apart from the
+  "normal" khronos validation.
   * Create a new JSON file with the new layer name.
-  * Set up the layer stack so that the "core_validation2" layer is on top of or before the standard validation
-  layer
-  * Then run tests and check for validation errors pointing to API usage in the "core_validation2" layer.
+  * Set up the layer stack so that the "khronos_validation2" layer is on top of or before the actual khronos
```
Comment (mikew-lunarg) [inline docs/gpu_validation.md]: capitalize Khronos?

File: layers/README.md [hunk 4]
```diff
@@ -36,33 +36,35 @@ Note that some layers are code-generated and will therefore exist in the directo
...
 
-### Layer Details
-For complete details of current validation layers, including all of the validation checks that they perform, please refer to the document `layers/vk_validation_layer_details.md`. Below is a brief overview of each layer.
-
 ### Standard Validation
-This is a meta-layer managed by the loader. (name = `VK_LAYER_LUNARG_standard_validation`) - specifying this layer name will cause the loader to load the all of the standard validation layers (listed below) in the following optimal order:  `VK_LAYER_GOOGLE_threading`, `VK_LAYER_LUNARG_parameter_validation`, `VK_LAYER_LUNARG_object_tracker`, `VK_LAYER_LUNARG_core_validation`, and `VK_LAYER_GOOGLE_unique_objects`. Other layers can be specified and the loader will remove duplicates.
+NOTE: This meta-layer is being deprecated -- users should load the Khronos validation layer (name = `VK_LAYER_KHRONOS_validation`) in its place.
+This is a meta-layer managed by the loader. (name = `VK_LAYER_LUNARG_standard_validation`) - specifying this layer name will cause the loader to load the standard validation layers:  `VK_LAYER_KHRONOS_validation`. Other layers can be specified and the loader will remove duplicates.
```
Comment (mikew-lunarg) [inline layers/README.md]: "load the standard validation layers" should be singular.

File: layers/README.md [hunk 5]
```diff
@@ -36,33 +36,35 @@ Note that some layers are code-generated and will therefore exist in the directo
...
-(build dir)/layers/object_tracker.cpp (name=`VK_LAYER_LUNARG_object_tracker`) - Track object creation, use, and destruction. As objects are created they are stored in a map. As objects are used the layer verifies they exist in the map, flagging errors for unknown objects. As objects are destroyed they are removed from the map. At `vkDestroyDevice()` and `vkDestroyInstance()` times, if any objects have not been destroyed they are reported as leaked objects. If a Dbg callback function is registered this layer will use callback function(s) for reporting, otherwise it will use stdout.
+The object lifetime tracking will track object creation, use, and destruction. As objects are created their handles are stored in a data structure. As objects are used the layer verifies they exist in the data structure and output errors for unknown objects. As objects are destroyed they are removed from the data structure. At `vkDestroyDevice()` and `vkDestroyInstance()` times, if any objects have not been destroyed they are reported as leaked objects. If a debug callback function is registered this layer will use callback function(s) for reporting, otherwise it will use stdout.
```
Comment (mikew-lunarg) [inline layers/README.md]: output -> "outputs" (or maybe "writes")

File: layers/README.md [hunk 6]
```diff
@@ -36,33 +36,35 @@ Note that some layers are code-generated and will therefore exist in the directo
...
...dation as states are created and used, and further validation Draw call and QueueSubmit time. Of primary interest is making sure that the resources bound to Descriptor Sets correctly align with the layout specified for the Set. Also, all of the image and buffer layouts are validated to make sure explicit layout transitions are properly managed. Related to memory, core\_validation includes tracking object bindings, memory hazards, and memory object lifetimes. It also validates several other hazard-related issues related to command buffers, fences, and memory mapping. Additionally core\_validation include shader validation (formerly separate shader\_checker layer) that inspects the SPIR-V shader images and fixed function pipeline stages at PSO creation time. It flags errors when inconsistencies are found across interfaces between shader stages. The exact behavior of the checks depends on the pair of pipeline stages involved. If a debug callback function is registered, this layer will use callback function(s) for reporting, otherwise uses stdout.  This layer also validates correct usage of image- and buffer-related APIs, including image and buffer parameters, formats, and correct use.
```
Comment (mikew-lunarg) [inline layers/README.md]: "further validation **at** Draw call"

File: docs/gpu_validation.md (hunk 1 above)
Comment (mark-lunarg) [reply docs/gpu_validation.md]: Thanks Mike.  I fixed the typos, and am inclined to leave 'khronos', when it describes the layer, as lower-case just as the other layer names were before.  In this case it isn't a proper noun, and in no instance is it spelled 'Khronos' when it's used -- either it'll be 'khronos' or 'KHRONOS'.  But my mind could be changed.

**Comment by mark-lunarg** [comment]
Fixed typos (except for one that I missed) and pushed to Master.

**Pull request closed without merging**